machines/aws/machine-set.yaml

```diff
@@ -51,8 +51,10 @@ spec:
               values:
               - "{{.AWS.ClusterName}}-worker-*"
           publicIp: true
+          {{- if .AWS.IAMInstanceProfile }}
```

Comment:
why not just always `id: "{{ .AWS.IAMInstanceProfile }}"`?

Reply:
Just thought it might make testing easier for now since it's totally possible to create these nodes with no profile, which means there's one less thing you have to set up in advance.

Reply:
Just hard coding the name still for now.